cmd/kubeadm/app/kubeadm.go

```diff
@@ -29,6 +29,8 @@ import (
 
 // Run creates and executes new kubeadm command
 func Run() error {
+	// To help debugging, immediately log version
+	glog.Infof("Version: %+v", version.Get())
```

Comment:
I was wondering why not using `kubeadm version` to get the version.

Reply:
I think that can be done too, or logging both

Reply:
if we are to add this it has to be a regular `fmt.Print*()`, because we want to keep the regular kubeadm output glog-free.

Reply:
on second though please remove the kubeadm change.
`kubeadm version` is sufficient output and there shouldn't be a top level printout for the version.

/hold